### PR TITLE
added performance fix

### PR DESCRIPTION
**Pull request by Eliasj42**
added performance fix, will need to import shark-runtime from my own fork unless my thread management functions are added to shark runtime

### CODE REVIEW

**Comment by powderluv** [comment]
Please commit to the SHARK-06xx2022 branch

**Comment by Eliasj42** [comment]
> 

Im not seeing that branch, can you send me a link?

**Comment by powderluv** [comment]
https://github.com/nod-ai/SHARK-Runtime/tree/shark-06032022

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "inference/thirdparty/shark-runtime"]
 	path = inference/thirdparty/shark-runtime
-	url = https://github.com/NodLabs/shark-runtime.git
+	url = https://github.com/Eliasj42/SHARK-Runtime-patch
```
Comment (powderluv) [inline .gitmodules:3]: please revert to nod-ai/shark-runtime

**Review by powderluv — COMMENTED** [review]
I think this .so needs to be built in the SHARK-Runtime/ repo and shipped as an artifact

**Comment by powderluv** [comment]
We can still merge this but the build of the .so can move to the runtime

**Pull request closed without merging**